src/common/Page.ts

```diff
@@ -605,6 +612,62 @@ export class Page extends EventEmitter {
     return this._javascriptEnabled;
   }
 
+  public async enableWebVitalsReporting(
+    enabledMetrics?: string[]
+  ): Promise<void> {
+    if (!isNode) {
+      throw new Error(
+        'Cannot enable Web Vitals reporting outside of Node.js environment.'
+      );
+    }
+
+    const fs = await helper.importFSModule();
+    const webVitalsScript = await fs.promises.readFile(
```

Comment:
@jackfranklin Do you have an idea how we could achieve this in a not Node.js specific way?